src/legacy/ui/ui_bundles/app_entry_template.js

```diff
@@ -26,7 +26,13 @@ export const appEntryTemplate = (bundle) => `
  * context: ${bundle.getContext()}
  */
 
-// import global polyfills before everything else
+// ensure the csp nonce is set in the dll
+import 'dll/set_csp_nonce';
```

Comment:
Question: Why does the dll setup stuff need to happen in the app entry file? Isn't that _not_ in a dll? Are you just being protective here in case `window.__webpack_nonce__` is unset before a dll is asynchronously loaded?

Reply:
This line is actually executing the code in the `dll/set_csp_nonce` module, which is inside the DLL, allowing us to mutate the internal nonce in the DLL before we load any other code. It is sorta defensive, it doesn't have to be the absolutely first thing we do, but I figure it's best to get it out of the way so that the DLL just works in the future.